py/farm_ng/canbus/packet.py

```diff
@@ -598,6 +598,6 @@ def from_raw_canbus_message(cls, message: canbus_pb2.RawCanbusMessage) -> BugDis
     def __str__(self):
         """Returns a string representation of the class."""
         return (
-            f"BugDispenserTpdo1: Rates: {self.rate0}, {self.rate1}, {self.rate2} "
+            f"BugDispenserTpdo3: Rates: {self.rate0}, {self.rate1}, {self.rate2} "
```

Comment:
This message is not a "tpdo3", it is a `BugDispenserState` message that happens to be sent on a `Tpdo3` channel

```suggestion
            f"BugDispenserState: Rates: {self.rate0}, {self.rate1}, {self.rate2} "
```